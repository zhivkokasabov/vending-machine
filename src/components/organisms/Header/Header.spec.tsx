import { render } from "@testing-library/react";
import { Header } from ".";

describe("Header tests", () => {
  const renderComponent = () => render(
    <Header />
  );

  it("should render", () => {
    const { asFragment } = renderComponent();

    expect(asFragment()).toMatchSnapshot();
  });
});
