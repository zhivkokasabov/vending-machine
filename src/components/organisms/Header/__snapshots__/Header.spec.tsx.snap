// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header tests should render 1`] = `
<DocumentFragment>
  <header
    class="bg-white shadow-md"
  >
    <div
      class="p-4 undefined mx-auto w-full max-w-[1240]"
    >
      <div
        class="text-2xl font-bold text-gray-800"
      >
        Vending Machine
      </div>
    </div>
  </header>
</DocumentFragment>
`;
